src/Builder.php

```diff
@@ -74,14 +74,14 @@ public function __construct(
     /**
      * Configures the audience
      *
-     * @param string $audience
+     * @param string|array $audience
      * @param boolean $replicateAsHeader
      *
      * @return Builder
      */
     public function setAudience($audience, $replicateAsHeader = false)
     {
-        return $this->setRegisteredClaim('aud', (string) $audience, $replicateAsHeader);
+        return $this->setRegisteredClaim('aud', $audience, $replicateAsHeader);
```

Comment:
Removing the type cast allows people to send anything to there which is not ideal.